packages/kernel-5.10/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "kernel-5_10"
+version = "0.1.0"
+edition = "2018"
+publish = false
+build = "build.rs"
+
+[package.metadata.build-package]
+package-name = "kernel-5.10"
+
+[lib]
+path = "pkg.rs"
+
+[[package.metadata.build-package.external-files]]
+# Use latest-srpm-url.sh to get this.
+url = "https://cdn.amazonlinux.com/blobstore/fa04b98fc067a4943beac60d0c2971e2fbef1a29faed4bac1c4096abe4ad4c12/kernel-5.10.29-27.126.amzn2.src.rpm"
+sha512 = "47341f4a1c13ba7e5ea72bad13fe689eefd22cc7547aea08a08fe47238b4a3fe1659786a406b84a1d1508143be20d9be2fae6fe3e7a6924bc85043bf61d4bfce"
```

Comment:
This seems to be missing the `[dependencies]` and `[build-dependencies]`. I would say all should be represented even if some are commented out.

Reply:
The kernel doesn't have any dependencies within the project, since it's a freestanding artifact that doesn't pull in any libraries.

I'll remove the `filesystem` dependency from the kernel.spec & Cargo.toml to help clarify this.

Reply:
Oh I see.